pfcpiface/grpcsim.go

```diff
@@ -222,19 +222,19 @@ func (u *upf) sim(mode simMode, s *SimModeInfo) {
 			fseID: uint64(n3TEID + i),
 			qfi:   9,
 			ulGbr: 50000,
-			ulMbr: 90000,
+			ulMbr: s.UplinkMBR,
```

Comment:
Do you not have to update `ulGbr`, as well? What happens if `ulMbr` is less than `ulGbr`?

Reply:
Typically uLGbr is for just one qos flow and has no relation with ulMbr. ulMbr covers non-gbr flows.

Reply:
Yes, I agree with you that it is for a single flow and the default values for simulating the flows is qfi 8 and 9 (i.e., non-GBR flows). However, my comment is more from the code perspective because, as you can see in the line above, `ulGbr` is defined/set even though it should not be used because the `qfi` is `9`. So, I wanted to make sure that the `ulGbr` is really ignored (i.e., not used). Additionally, given that the flows here are non-GBR (qfi 8 and 9), I do not know why the `ulGbr` was assigned. Maybe they were planning to make these parameters flexible such that a `qfi` (`5qi`) for GBR flow can be assigned?

Reply:
FYI, I did tested the traffic with UDP and changed the ulGbr/dlGbr values to different values and confirmed that it has no impact.